lexik/jwt-authentication-bundle/2.3/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+    "bundles": {
+        "Lexik\\Bundle\\JWTAuthenticationBundle\\LexikJWTAuthenticationBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "etc/": "%ETC_DIR%"
+    },
+    "env": {
+        "JWT_PRIVATE_KEY_PATH": "%kernel.project_dir%/var/jwt/private.pem",
```

Comment:
I explained myself wrong. The var/ dir still exists in Flex apps ... but you can't use `%VAR_DIR%` in your recipes because that variable is not defined. But if we define variables for everything (bin/, etc/, src/, web/), why shouldn't we provide it for var/ too? I've proposed to add this in the Flex repo (which is still private, that's why I don't link it here).